### PR TITLE
added core module and empty InvoiceProvider

### PR DESCRIPTION
**Pull request by KrzKos**
if everything is fine, please approve.

### CODE REVIEW

File: fakturogen-core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	<name>fakturogen-core</name>
+	<description>Core for fakturogen</description>
+
+	<properties>
+		<java.version>11</java.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
```
Comment (ewa-git) [inline fakturogen-core/pom.xml:23]: I think we  should have only one dependency for springboot and springboot  test parent module

**Review by ewa-git — APPROVED** [review]
see my comments

File: fakturogen-core/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
```
Comment (rowinskidamian) [inline fakturogen-core/pom.xml:6]: Current parent is for main-root module. This parent should be here:
<parent>
        <artifactId>fakturogen-parent</artifactId>
        <groupId>pl.fakturogen</groupId>
        <version>1.0-SNAPSHOT</version>
    </parent>

**Review by rowinskidamian — CHANGES_REQUESTED** [review]
one comment regarding pom.xml in module

File: fakturogen-core/pom.xml (hunk 1 above)
Comment (KrzKos) [reply fakturogen-core/pom.xml:23]: Deleted

File: fakturogen-core/pom.xml (hunk 2 above)
Comment (KrzKos) [reply fakturogen-core/pom.xml:6]: Agree.

**Review by rowinskidamian — APPROVED** [review]
alles gut 👍

**Pull request merged**